client/signup/main.jsx

```diff
@@ -106,7 +105,7 @@ const Signup = React.createClass( {
 					: undefined;
 				const filteredDestination = utils.getDestination( destination, dependencies, this.props.flowName );
 				let loadingScreenDelay;
-				if ( dependencies.cartItem && abtest( 'signupCheckoutRedirect' ) === 'auto' ) {
+				if ( dependencies.cartItem ) {
 					loadingScreenDelay = MINIMUM_TIME_LOADING_SCREEN_IS_DISPLAYED_PREMIUM;
```

Comment:
Can we refactor this bit? You had added two constants for a minimum loading time so we could target the "paid user" case and leave the "free user" case as-is. But I think we may as well just go back to one constant and change it to three seconds.